SoftLayer/managers/storage.py

```diff
@@ -577,3 +577,12 @@ def convert_dep_dupe(self, volume_id):
         return self.client.call('Network_Storage',
                                 'convertCloneDependentToIndependent',
                                 id=volume_id)
+
+    def split_percentage(self, volume_id):
+        """Get the Clone split/move status completion of a duplicate volume
```

Comment:
please put the link reference

Reply:
ahh.. which link reference to you are asking here...jira for API that's consuming this - https://jira.softlayer.local/browse/FBLOCK-4520